src/uct/ib/base/ib_iface.c

```diff
@@ -392,7 +393,8 @@ static ucs_status_t uct_ib_iface_init_lmc(uct_ib_iface_t *iface,
 }
 
 static ucs_status_t uct_ib_iface_create_cq(uct_ib_iface_t *iface, int cq_length,
-                                           size_t inl, struct ibv_cq **cq_p)
+                                           size_t inl, struct ibv_cq **cq_p,
+                                           int cq_comp_vector)
```

Comment:
This variable name is cryptic a bit. Can it contain something else than a cpu_set mask?